tests/AppBundle/Committee/Serializer/CommitteeEventICalHandlerTest.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
You don't need to put this on top of each file, Symfony already transform type errors into exceptions.